Documentation/configuration/metrics.rst

```diff
@@ -50,6 +53,8 @@ Policy Imports
 --------------
 
 * ``policy_count``: Number of policies currently loaded
+* ``policy_regeneration_seconds``: Total time taken to regenerate policies
```

Comment:
You must also add a counter of the number of policy regenerations.

Reply:
We can already derive that value using `policy_count` i.e `Number of policies currently loaded` and `policy_import_errors` i.e Number of times a policy import has failed. I felt adding a third counter would be redundant.

Reply:
let me know your thoughts.

Reply:
@rlenglet Please let me know if I need to have additional counter here?